deployments/automated-previews.mdx

```diff
@@ -151,59 +151,75 @@ jobs:
 
 ## Gitlab CI
 
-This job can be pasted into your `.gitlab-ci.yml` at the root of your repository. You are welcome to change the `stage` to whatever fits your needs to allow you to run tests before the preview is generated, and please be sure to assign correct values for the variables in the job. Additionally, you'll need to assign values for variables in the below config not prefixed with `$CI_` in your repository's CI variables configuration so that the architect commands will run successfully.
+### Creating and cleaning up preview environments
 
-This configuration takes advantage of GitLab environments in order to give you better control and visibility into what environments exist and what's deployed to them. On PR creation, both a GitLab and Architect environment will be created. The component specified in the repository will be registered with the Architect Cloud and deployed to the environment. When the PR is either merged or closed, the GitLab environment will be automatically deleted and the component deployed to the environment in the Architect Cloud will be destroyed.
+This job can be pasted into your `.gitlab-ci.yml` at the root of your repository. You are welcome to change the `stage` to whatever fits your needs to allow you to run
+tests before the preview is generated. Please be sure to assign correct values for the variables in the job marked with `<set-this-as-a-CI/CD-variable>` as well as the
+ones mentioned in the warning below.
+
+This configuration takes advantage of GitLab environments in order to give you better control and visibility into what environments exist and what's deployed to them.
+On PR creation, both a GitLab and Architect environment will be created. The component specified in the repository will be registered with the Architect Cloud and
+deployed to the environment. When the PR is either merged or closed, the GitLab environment will be automatically deleted and the component deployed to the environment
+in the Architect Cloud will be destroyed.
+
+<Warning>
+  This example assumes that the repo has `ARCHITECT_EMAIL`, `ARCHITECT_PASSWORD`, `ARCHITECT_ACCOUNT`, and `ARCHITECT_CLUSTER` set as CI/CD variables.
+</Warning>
 
 <Warning>
   `ARCHITECT_PASSWORD` must be a [personal access
   token](https://cloud.architect.io/login).
```

Comment:
Similar comment for the BitBucket - we want to ensure the user knows what the PAT is and how to generate one

Reply:
https://gitlab.com/architect-io/product/-/issues/281

Reply:
Updated to include that link